tests/scripts/kubeadm-install.sh

```diff
@@ -5,6 +5,8 @@ KUBE_VERSION=${1:-"v1.7.5"}
 null_str=
 KUBE_INSTALL_VERSION="${KUBE_VERSION/v/$null_str}"-00
 
+#need to disable swap when installing k8s 1.8.x via kubeadm
```

Comment:
You might want to provide a link to the issue. https://github.com/kubernetes/kubernetes/issues/34726